### ISSUE TITLE
python version mismatch

### ISSUE THREAD

**Issue by skunkwerk** [TimoBolkart/voca#2]
This repo requires python2.7, but DeepSpeech (a dependency) requires python 3.  Any thoughts on how to reconcile the two?  

I tried installing voca & mesh with python 3, but mesh only works with python 2.7.

thank you

**Comment by ak9250** [comment]
i think I am getting a related problem to this 

Traceback (most recent call last):
  File "/content/voca/visualize_sequence.py", line 23, in <module>
    from psbody.mesh import Mesh
ModuleNotFoundError: No module named 'psbody'

**Comment by TimoBolkart** [comment]
> This repo requires python2.7, but DeepSpeech (a dependency) requires python 3. Any thoughts on how to reconcile the two?
> 
> I tried installing voca & mesh with python 3, but mesh only works with python 2.7.
> 
> thank you

You are right about the mesh package which only works for Python 2.7. This being said, DeepSpeech actually works for Python 2.7 as well. Just download the deepspeech-0.1.0-models.tar.gz from https://github.com/mozilla/DeepSpeech/releases/tag/v0.1.0 and specify the path of the trained DeepSpeech model when calling the VOCA code as done in the sample code.

**Comment by TimoBolkart** [comment]
> i think I am getting a related problem to this
> 
> Traceback (most recent call last):
> File "/content/voca/visualize_sequence.py", line 23, in
> from psbody.mesh import Mesh
> ModuleNotFoundError: No module named 'psbody'

Did you download and successfully install the mesh package (https://github.com/MPI-IS/mesh) within the virtual environment?

**Comment by ak9250** [comment]
@TimoBolkart I am trying to get this working in google colab, will try again

**Comment by ak9250** [comment]
@TimoBolkart is it possible to do something like this https://www.youtube.com/watch?v=9Yq67CjDqvw instead of animating a 3d mesh?

**Comment by TimoBolkart** [comment]
@ak9250 To get a realistic looking 2D animation, one could potentially combine the animated 3D mesh with some texture and a neural renderer [https://arxiv.org/pdf/1904.12356.pdf] to increase realism.

**Comment by skunkwerk** [comment]
thanks @TimoBolkart, I managed to get it working - as you mentioned, you don't actually need to install DeepSpeech - just the pre-built binary models that you link to.  I was able to get voca working on a Google Cloud virtual machine with a GPU using the Intel® optimized Deep Learning Image: TensorFlow 1.13.1 m25 (with Intel® MKL-DNN/MKL and CUDA 10.0).

The tricky part is actually installing the mesh libraries - if your machine has both Python 2 & Python 3 installed, it will always try to create a Python 3 virtual environment and fail.  My fix was to do this from the shell:
export VIRTUALENV_PYTHON=/usr/bin/python2.7

thanks!
